src/helm/benchmark/scenarios/viet_wvs_scenario.py

```diff
@@ -0,0 +1,209 @@
+    
+    def get_few_shot_examples(self) -> str:
+        """Return a few-shot learning examples formatted as a string."""
+        examples = """
+            Here are some examples:
+
+            Example 1:
+            You are a 35 year old male from Vietnam. You have higher education and consider yourself middle class. You are married.
+
+            How important is family in your life?
+
+            1. Very important
+            2. Rather important
+            3. Not very important
+            4. Not at all important
+
+            Please select the option number that best represents your view.
+            1
+
+            Example 2:
+            You are a 28 year old female from Vietnam. You have middle education and consider yourself working class. You are single.
+
+            Would you say that most people can be trusted or that you need to be very careful in dealing with people?
+
+            1. Most people can be trusted
+            2. Need to be very careful
+
+            Please select the option number that best represents your view.
+            2
+
+            Example 3:
+            You are a 50 year old male from Vietnam. You have lower education and consider yourself lower class. You are married.
+
+            Do you think that homosexuality is justifiable?
+
+            1. Never justifiable
+            2. Rarely justifiable
+            3. Sometimes justifiable
+            4. Always justifiable
+
+            Please select the option number that best represents your view.
+            1
+
+            Now answer the following question:
+
+        """
+        return examples
+    
+    def get_instances(self, output_path: str) -> List[Instance]:
+        """Generate test instances from Vietnam personas and WVS questions."""
+        instances: List[Instance] = []
+        
+        try:
+            # Download files from Hugging Face Hub
+            personas_path = hf_hub_download(
+                repo_id=self.repo_id, 
+                filename=self.personas_filename,
+                repo_type="dataset"
+            )
+            
+            questions_path = hf_hub_download(
+                repo_id=self.repo_id, 
+                filename=self.questions_filename,
+                repo_type="dataset"
+            )
+            
+            # Load the downloaded files
+            with open(personas_path, "r", encoding="utf-8") as f:
+                personas = json.load(f)
+            
+            with open(questions_path, "r", encoding="utf-8") as f:
+                questions = json.load(f)
+                
+        except Exception as e:
+            print(f"Error loading required files from Hugging Face: {e}")
+            return []
+
+        # Get few-shot examples
+        few_shot_examples = self.get_few_shot_examples() if self.include_examples else ""
+        
+        # Sample personas
+        sampled_personas = random.sample(personas, min(self.num_personas, len(personas)))
```

Comment:
I would recommend letting HELM do the evaluation example sampling rather than doing it yourself.

If you really want to do the sampling yourself, you should fix the random seed so that the sampling is deterministic.